modern/clang/Dockerfile

```diff
@@ -121,9 +122,9 @@ RUN apt-get -qq update \
     && curl -s -O "${ARTIFACTORY_REPOSITORY}/gcc-${LIBSTDCPP_VERSION}.tar.gz" \
     && tar zxf gcc-${LIBSTDCPP_VERSION}.tar.gz -C /tmp/gcc
 
-FROM ${DOCKER_USERNAME}/base-ubuntu16.04 as deploy
+FROM ${DOCKER_USERNAME}/base-ubuntu16.04:${CONAN_VERSION} as deploy
```

Comment:
For each new Conan release, we only need to rebuild the deploy stage. If we put Base:tag as base, it will require rebuilding Base for every new Conan release too. It takes more time. That's why I preferred updating Conan in Deploy stage. Actually, we don't need Conan installed in Base, I installed in case someone needs Conan in a raw image without compiler, but I think is same case as vim, you should install only when need, not for every image.